src/components/Tooltip.jsx

```diff
@@ -43,6 +44,10 @@ export const UserTooltip = ({ user, ...props }) => {
     other_person_id: user.id,
   });
 
+  const redirectToModlogActedOn = (user) => {
+    navigate(`/actions?acted_on__id=${user.id}`);
```

Comment:
double underscore

Reply:
Ohh yeah i oversaw that one. ( I copied your text in the issue ) I fixed that one in the mod logs but not there